ScnSoft-Education/LeetCodeTraining/LongestCommonPrefix/LongestCommonPrefixSolution.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text;
+
+namespace LeetCodeTraining.LongestCommonPrefix
+{
+    //https://leetcode.com/problems/longest-common-prefix/
+    public static class LongestCommonPrefixSolution
+    {
+        public static string LongestCommonPrefix(string[] strs)
+        {
+            StringBuilder prefix = new StringBuilder();
+            string shortestWord = strs.ToList().OrderBy(x => x.Length).First();
```

Comment:
Please remove invoking extra extension ToList() and update after that the LC screenshot wit up to date solution.